installers/install.sh

```diff
@@ -58,14 +57,41 @@ run() {
         native_loc="https://github.com/tridactyl/native_messenger/releases/download/$native_version/native_main-$binary_suffix"
     fi
 
-    mkdir -p "$manifest_home" "$XDG_DATA_HOME"
+    install_to "$manifest_home" "$manifest_home"
 
-    manifest_file="$manifest_home/tridactyl.json"
-    native_file="$XDG_DATA_HOME/native_main"
+    for flatpak_dir in ~/.var/app/*/.mozilla; do
```

Comment:
I'm a bit nervous about running this on OSX - could you set a flat around line 36 to skip it on them?